package.json

```diff
@@ -30,6 +30,9 @@
     }
   },
   "dependencies": {
+    "carbon-components": "9.66.3",
```

Comment:
we want these only in dev-dependencies

Reply:
oop, thanks for this. was likely a leftover from the yarn issues we were having during our pairing session